src/test/unit/common/error/integrations/onUncaughtException.test.ts

```diff
@@ -0,0 +1,50 @@
+import { strictEqual } from 'assert';
+import { OnUncaughtException } from '../../../../../snyk/common/error/integrations/onUncaughtException';
+
+suite('ErrorReporter OnUncaughtException Integration', () => {
+  let uncaughtExceptionIntegration: OnUncaughtException;
+
+  setup(() => {
+    uncaughtExceptionIntegration = new OnUncaughtException({
+      extensionPath: '/Users/snyk/.vscode/extensions/snyk-security.snyk-vulnerability-scanner-1.2.4',
+    });
+  });
+
+  test('Is extension origin error', () => {
+    const error = new Error();
+    error.stack = `Error: test error
+    at Test.Runnable.run (/Users/snyk/.vscode/extensions/snyk-security.snyk-vulnerability-scanner-1.2.4/out/extension.js:100:5)`;
+
+    const res = uncaughtExceptionIntegration.isExtensionOriginError(error);
+
+    strictEqual(res, true);
+  });
+
+  test('Is not extension origin error - another extension stacktrace', () => {
+    const error = new Error();
+    error.stack = `Error: test error
+    at Test.Runnable.run (/Users/snyk/.vscode/extensions/microsoft.sample-extension-1.2.4/out/extension.js:100:5)`;
```

Comment:
Maybe also test with Windows/Linux paths?

Reply:
fair point, did want to cover just the basics from now. I've added other paths now (Linux and MacOS have both the same extension folder path).